blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -56,29 +53,26 @@ The meeting room behavior of Gather is fundamentally different to the rest of th
 ### Random bumps
 When you walk to someone, while approaching, the video and audio slowly fade in. That technique is called "spatial audio technology" and mimics the way we bump into other people in real life, where someone's voice is louder while you are near them, and softer as you walk away. This is wonderful because it feels so familiar and natural.
 
-<video preload autoplay loop width="100%">
-  <source src="images/gather/video.mp4" type="video/mp4" />
-  <source src="images/gather/video.mp4" type="video/mp4" />
-</video>
-
-_People bumping into each other (via [gather.town](http://www.gather.town))_
+<figure>
+    <video preload autoplay loop width="100%">
+      <source src="images/gather/video.mp4" type="video/mp4" />
+      <source src="images/gather/video.mp4" type="video/mp4" />
+    </video>
+    <figcaption>People bumping into each other (via <a href='http://www.gather.town'> gather.town</a>)</figcaption>
+</figure>
 
 
 ### Pair programming
 A similar effect happens when you sit in the virtual office and suddenly two folks group up on the other side of the table. Instantly you know, _"oh, they do some pair programming"_. When I asked my colleagues about their session, they said it's a nice way to meet at some actual (although virtual) location and then just _"bump into the other person"_ to start the session. 
 
 This feeling is the promise of the Metaverse. It brings back the informal human interactions and spontaneity. You don't have to create a video meeting upfront or send some specific invitation. Who would do that in real life? Instead, you just walk over to the desk of your colleague and start talking.  
 
-![](images/gather/gather-pair.gif)
-
-_Our colleagues Jesús & Felix having a pair programming session_
+![Our colleagues Jesús & Felix having a pair programming session](images/gather/gather-pair.gif)
```

Comment:
This gif doesn't have a caption. Is there something broken?

Reply:
@feedm3 thanks for the feedback 🙏 
Gifs and videos are not processed by `gatsby-remark-images`, but by `gatsby-remark-copy-linked-files`. That's why the captions are not displayed. As a workaround, you can include `<figcaption />` explicitly in the markdown. If you think of a better solution, please let me know.